src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -25,7 +27,6 @@ object SlickDataAccess {
   type IoValue = String
   val IoInput = "INPUT"
   val IoOutput = "OUTPUT"
-  val IndexNone = -1 // "It's a feature" https://bugs.mysql.com/bug.php?id=8173
```

Comment:
It was moved to ExecutionIndex to have everything related to this at the same place:
```scala
 object ExecutionIndex {
    type ExecutionIndex = Option[Int]
    val IndexNone = -1 // "It's a feature" https://bugs.mysql.com/bug.php?id=8173
```